index.html

```diff
@@ -82,7 +85,9 @@
             </div>
         </div>
     </div>
-
+        
+<iframe src="https://calendar.google.com/calendar/embed?title=MountSion&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;showTz=0&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=khanrahan%40mountsioncbssecondary.ie&amp;color=%231B887A&amp;ctz=Europe%2FDublin" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

Comment:
Can you move this into the `4 wide column` under the twitter feed, and then upload a screenshot of it to the PR, just to give an indication of what it would look like in the sidebar, I wouldn't be a fan of its current placement on the page.